Capture the DX, VPN and Peering connections for all VPCs

jchrisfarris opened the pull request:
Please pay extra attention to both how I structured this json and the fact I'm putting it in the unused VPC section of the database.

Commit:
Capture the DX, VPN and Peering connections for all VPCs

SpenGietz reviewed (changes requested):
A few changes requested. Solid module 👍

Commit:
Feedback for PR56

BenFriedland-Rhino committed:
Adapt enum_lateral_movement database layout

This commit changes the enum_lateral_movement module's database layout
to match the current format of PacuSession.VPC data.

BenFriedland-Rhino committed:
Ensure VPC entity subkeys can't be set to null

BenFriedland-Rhino committed:
Store VPC peerings and DX gateways inside VPCs

BenFriedland-Rhino committed:
Remove the obsolete help function
